learn/wavemaker-release-notes/v11-5-3.md

```diff
@@ -0,0 +1,64 @@
+### React Native
+
+- Fixed the issue where the text in the Label widget is not center-aligned even if the text alignment is set as center.
+- Fixed the issue where the next, prev, cancel, done and skip methods were not exposed in the Wizard widget.
+- Fixed the customization issue by adding the active class in the Checkboxset and RadioSet widgets.
+- Fixed navigation issue by adding support to page links and empty links in the Label widget.
+- Fixed the issue where the Prefab.onPropertyChange was not getting triggered on loading the page.
+- Fixed the customization issue by supporting the application of custom color to each bar in the Column chart widget.
+- Fixed the issue where the background image gets extended along with the page content till the bottom of the page when the page-level scroll bar is added.
+- Fixed navigation issue in the Wizard widget by adding the on-change event.
+- Fixed the issue in Tabs where the Label widget took more than 100% of size when added inside the List widget. 
+- Fixed the issue in IPA or APK files where a user encountered an error while using the tabs in a page.
+- Fixed the Stack bar chart issue where a user could see a circle with a straight line with a dot in the center or white space above the graph if the dataset contains negative values.
+- Fixed the issue where the users were unable to add or customize the toast message styles shown for Notification Actions.
+- Fixed Form validation issue where the `Validatefieldsonsubmit` method did not show true until the form was selected and the save button was clicked.
+- Fixed an issue in the Checkboxset widget where only one value is getting displayed as the displayValue even when multiple values are selected.
+- Fixed multiple issues such as being unable to input data in Form fields and Text widget in a list, default values not getting displayed in Form fields and Number widget.
+- Fixed the issue where the user was able to view the same data values from props.datavalue and onchange events.
+- Fixed the navigation issue where the user was directed from one step to another even when the condition failed.
+- Fixed the issue where the user can now access the widgets when the On Open event of a dialog is triggered.
+- Fixed the issue in the Number widget where string values are displayed as output on tapping the Reset button.
+- Fixed the Datetime widget issue where the user was unable to set the timezone.
+- Fixed the issue where the user was redirected to login issue when 401 is thrown from imported web service for a logged-in user.
+- Fixed Label issue where List template must have a property to set ellipsis to text when text is multiline.
+
+
+### Studio Page Workspace
+
+- Fixed the issue where the user was unable to view the hidden tab content.
+- Fixed the issue in the Alert dialog where the button and content were not displaying correctly compared to the previous version.
+- Fixed the issue where the tab panes on the partial AMAssetForm were not visible.
```

Comment:
Do not mention names used by customers in their projects. Check with the dev/qa on a generic issue description and mention the same.